.swiftlint.yml

```diff
@@ -9,6 +9,8 @@ disabled_rules: # rule identifiers to exclude from running
   - control_statement # no rounded brackets around conditions. we would need this the other way around
   - nesting	# func nesting max 1 level
   - todo # TODO can be written in the code but should be linked to a ticket on JIRA.
+  - cyclomatic_complexity # our switch statements make it impossible to follow this rule (https://en.wikipedia.org/wiki/Cyclomatic_complexity)
+  - function_parameter_count # only 5 function parameters are allow. Our APIManager usually requires more.
```

Comment:
I actually believe that this one is a very good thing and 5 parameters are more than enough.